src/test/resources/core-case-data/search-response-with-def.success.json

```diff
@@ -0,0 +1,372 @@
+{
+  "id": 1516189555935242,
+  "jurisdiction": "CMC",
+  "state": "claimIssued",
+  "case_type_id": "MoneyClaimCase",
+  "created_date": "2018-01-17T11:45:55.944",
+  "last_modified": "2018-01-17T11:45:55.989",
+  "security_classification": "PUBLIC",
+  "case_data": {
+    "referenceNumber": "000MC023",
+    "submitterId": "20",
+    "submittedOn": "2018-01-17T11:45:46.872525",
+    "responseDeadline": "2018-01-17T11:45:55.944",
+    "defendantId": "100",
```

Comment:
To be more real we should have letter holder ID at this point of case lifecycle as well. Not necessary for the tests but in the future when someone wants to reuse the JSON object it might come as a surprise that letter holder is missing.